tests/utils_tests/test_datastructures.py

```diff
@@ -115,6 +165,77 @@ def test_getlist_none_empty_values(self):
         self.assertIsNone(x.getlist('a'))
         self.assertEqual(x.getlist('b'), [])
 
+    def test_setitem(self):
+        x = MultiValueDict({'a': [1, 2]})
+        x['a'] = 3
+        self.assertEqual(list(x.lists()), [('a', [3])])
```

Comment:
Maybe:
```suggestion
        self.assertSequenceEqual(x.lists(), [('a', [3])])
```
I think we can use the same in below tests.

Reply:
That's a no-go:
```
======================================================================
FAIL: test_setitem (utils_tests.test_datastructures.MultiValueDictTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".../django/tests/utils_tests/test_datastructures.py", line 162, in test_setitem
    self.assertSequenceEqual(x.lists(), [('a', [3])])
AssertionError: First sequence has no length.    Non-sequence?
- <dict_itemiterator object at 0x7f13abd19900>
+ [('a', [3])]
```

Reply:
OK, it was just a shot in the dark :dart: